### ISSUE TITLE
Add-DbaServerRoleMember and Remove-DbaServerRoleMember

### ISSUE THREAD

**Issue by JoelMiller74** [sqlcollaborative/dbatools#7681]
### Summarize Command's Functionality'

It appears there are commands to add and remove Server roles, add and remove DB roles, and add/remove members from a DB role. However, there are no commands to add/remove members from server roles. I am trying to automate everything in our SQL Server cookbooks (we use Chef) using DBATools and feel this one is critical to get automated.
Thanks!

### Is there a command that is similiar or close to what you are looking for?

Yes

### Technical Details

similar commands:
https://docs.dbatools.io/#Add-DbaDbRoleMember
https://docs.dbatools.io/#Remove-DbaDbRoleMember

**Comment by MikeyBronowski** [comment]
Have you tried this one for adding members to the server role?
https://docs.dbatools.io/Add-DbaServerRoleMember

**Comment by JoelMiller74** [comment]
No. I did not know that existed. I look for the list of all commands on the commands page: https://dbatools.io/commands/
Then, I look for syntax on the docs page: https://docs.dbatools.io/
It appears that Add-DbaServerRoleMember is not on the commands page, but is on the docs page.
However, it appears that Remove-DbaServerRoleMember is missing from both.
Thanks!

**Comment by MikeyBronowski** [comment]
@JoelMiller74  you're right about the Remove-* function, it's not in the module.

@potatoqualitee can you please remind me what is the best way to update the  https://dbatools.io/commands/ page content? In the past it was WordPress, but don't think I made any changes since the migration.

**Comment by potatoqualitee** [comment]
Thanks, @MikeyBronowski! The website is severely out of date until I figure out how to automatically update it in our publish process. Till then, it can be updated here: https://github.com/sqlcollaborative/web/tree/html/commands
